src/content/guides/farmlands-pos-integration.mdoc

```diff
@@ -382,41 +386,48 @@ autonumber
 
 participant Cardholder
 participant POS
+participant POS Server
 participant Centrapay
 
 Note over Cardholder: Present Barcode
 Cardholder->>POS: Scan Barcode
 
-POS->>Centrapay: Create Payment Request with Barcode and the preAuth flag
+POS->>POS Server: Request Payment
+
+POS Server->>Centrapay: Create Payment Request with Barcode and the preAuth flag
 
 loop Poll for Payment Confirmation
-	POS->>Centrapay: Get Payment Request
+	POS Server->>Centrapay: Get Payment Request
 
-	alt Payment Condition Pending
-			note over POS: ❓ Prompt Cashier to Accept or Decline Condition
-			POS->>Centrapay: POS Accepts Condition
-		else Payment Request Status is paid
-			Note over POS: Stop Polling for Confirmation
-		end
+  opt Payment Condition Pending
+    note over POS: ❓ Prompt Cashier to Accept or Decline Condition
+    POS->>Centrapay: POS Accepts Condition
```

Comment:
Change: In your other diagram you have this POS server -> Centrapay. Change to make it consistent to go POS -> POS Server -> Centrapay.